src/main/java/lesson01/arraysmanipulation/ArrayTypesConvertor.java

```diff
@@ -0,0 +1,23 @@
+package lesson01.arraysmanipulation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayTypesConvertor {
+
+    static int[] arrayListToIntArrayConversion (List<Integer> inputArray){
+        int[] convertedArray = new int[inputArray.size()];
+        for (int i = 0; i < inputArray.size(); i++){
+            convertedArray[i] = inputArray.get(i);
+        }
+        return convertedArray;
+    }
+
+    public static List<Integer> intArrayToListArrayConversion (int[] inputArray){
```

Comment:
I think this name is more consistent intArrayToArrayListConversion

Reply:
agree. changed